examples/retinanet/train_single_thread.py

```diff
@@ -0,0 +1,436 @@
+from flax.training import checkpoints
```

Comment:
Where is this file used?
Could you add some file-level pydoc explaining the differences between this and `train.py` ?

Reply:
@DanGraur This was used for debugging, do you still need it?

Reply:
Indeed, this is only for debugging. I will remove this source.